pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -235,17 +235,20 @@ const fluentConfTemplate = `{{- define "fluentConf" }}
 		<elasticsearch_index_name>
 			enabled "#{ENV['ENABLE_ES_INDEX_NAME'] || 'true'}"
 			tag "journal.system** system.var.log** **_default_** **_kube-*_** **_openshift-*_** **_openshift_**"
-			name_type operations_full
+			name_type static
+			static_index_name infra-write
 		</elasticsearch_index_name>
 		<elasticsearch_index_name>
 			enabled "#{ENV['ENABLE_ES_INDEX_NAME'] || 'true'}"
 			tag "linux-audit.log** k8s-audit.log** openshift-audit.log**"
-			name_type audit_full
+			name_type static
+			static_index_name audit.infra-write
```

Comment:
is there a fluentd change that needs to go along with this?

Reply:
yes. ref'd in the original comment: https://github.com/openshift/origin-aggregated-logging/pull/1797